source/vibe/data/json.d

```diff
@@ -44,7 +44,7 @@ unittest {
 public import vibe.data.serialization;
 
 public import std.json : JSONException;
-import std.algorithm : equal, min;
+import std.algorithm : equal, min, max;
```

Comment:
`/tests/redis/source/app.d` used equal from `json`. I've added an import to app.d